packages/adapter-auto/tsconfig.json

```diff
@@ -7,7 +7,8 @@
 		"target": "es2022",
 		"module": "node16",
 		"moduleResolution": "node16",
-		"baseUrl": "."
+		"baseUrl": ".",
+		"skipLibCheck": true
```

Comment:
Do we need `skipLibCheck`? I've tried running `pnpm check` without it and didn't bump into any issues for the adapters.

Reply:
ah nice. I don't have time to review that PR right now but I left a comment over there to remind myself to come back and clean this up later